docs/source/tutorials/llama2-13b-chatbot.mdx

```diff
@@ -0,0 +1,251 @@
+
+This model is part of the **Llama 2** family of models, and has been tuned to recognize chat interactions
+between a *user* and an *assistant* (more on that later).
+
+As explained in the [optimum-neuron documentation](https://huggingface.co/docs/optimum-neuron/guides/export_model#why-compile-to-neuron-model)
+, models need to be compiled and exported to a serialized format before running them on Neuron devices.
+
+Fortunately, 🤗 **optimum-neuron** offers a [very simple API](https://huggingface.co/docs/optimum-neuron/guides/models#configuring-the-export-of-a-generative-model)
+to export standard 🤗 [transformers models](https://huggingface.co/docs/transformers/index) to the Neuron format.
+
+When exporting the model, we will specify two sets of parameters:
+
+- using *compiler_args*, we specify on how many cores we want the model to be deployed (each neuron device has two cores), and with which precision (here *float16*),
+- using *input_shapes*, we set the static input and output dimensions of the model. All model compilers require static shapes, and neuron makes no exception. Note that the
+*sequence_length* not only constrains the length of the input context, but also the length of the Key/Value cache, and thus, the output length.
+
+Depending on your choice of parameters and inferentia host, this may take from a few minutes to more than an hour.
+
+For your convenience, we host a pre-compiled version of that model on the Hugging Face hub, so you can skip the export and start using the model immediately in paragraph 2.
+
+
+```python
+from optimum.neuron import NeuronModelForCausalLM
+
+compiler_args = {"num_cores": 24, "auto_cast_type": 'fp16'}
+input_shapes = {"batch_size": 1, "sequence_length": 2048}
+model = NeuronModelForCausalLM.from_pretrained(
+        "NousResearch/Llama-2-13b-chat-hf",
+        export=True,
+        **compiler_args,
+        **input_shapes)
+```
+
+This will probably take a while.
+
+Fortunately, you will need to do this only once because you can save your model and reload it later.
+
+
+```python
+model.save_pretrained("llama-2-13b-chat-neuron")
+```
+
+Even better, you can push it to the [Hugging Face hub](https://huggingface.co/models).
+
+For that, you need to be logged in to a [HuggingFace account](https://huggingface.co/join).
+
+In the terminal, just type the following command and paste your Hugging Face token when requested:
+
+```shell
+huggingface-cli login
+```
+
+By default, the model will be uploaded to your account (organization equal to your user name).
+
+Feel free to edit the code below if you want to upload the model to a specific [Hugging Face organization](https://huggingface.co/docs/hub/organizations).
+
+
+```python
+from huggingface_hub import whoami
+
+org = whoami()['name']
+
+repo_id = f"{org}/llama-2-13b-chat-neuron"
+
+model.push_to_hub("llama-2-13b-chat-neuron", repository_id=repo_id)
```

Comment:
we could add the bs and seq length in the repo to make it easier for people to know

Reply:
These parameters are in `config.json` under the `neuron` subconfig. You mean generating a model card automatically in push_to_hub ?